apps/src/javalab/JavalabSettings.jsx

```diff
@@ -1,28 +1,33 @@
 import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
 import i18n from '@cdo/locale';
+import javalabMsg from '@cdo/javalab/locale';
```

Comment:
nit: it's a little confusing that some strings are `i18n` and others are `javalabMsg` -- maybe `javalabI18n` instead?

Reply:
I think we tend to use `javalabMsg` across most Javalab components, so I can change `i18n` to `msg` instead if that's clearer?

Reply:
works for me!